### ISSUE TITLE
Add the option to overlay the display of mapped keys

### ISSUE THREAD

**Issue by Racansky** [barry-ran/QtScrcpy#361]
It would be great to be able to overlay the display with the mapped key pointer and their position.
Just like various emulators do.

**Comment by barry-ran** [comment]
Indeed, the main reason for not doing this is that it takes too long, I will try my best

**Comment by guanzhangrtk** [comment]
I thought this feature is already implemented, so is this image just a mockup?

https://github.com/barry-ran/QtScrcpy/blob/master/screenshot/game.jpg

**Comment by barry-ran** [comment]
yes




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "barry-ran/QtScrcpy"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年6月4日(星期五) 中午1:20
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [barry-ran/QtScrcpy] Add the option to overlay the display of mapped keys (#361)





 
I thought this feature is already implemented, so is this image just a mockup?
 
https://github.com/barry-ran/QtScrcpy/blob/master/screenshot/game.jpg

**Comment by guanzhangrtk** [comment]
Thanks for the quick response, I hope this can be implemented in the future!

**Comment by guanzhangrtk** [comment]
Why was this issue closed? Is it because it is already implemented or you don't plan to implement? Thanks!
